src/main/java/seedu/address/model/student/Student.java

```diff
@@ -33,15 +35,18 @@ public class Student extends Person {
     // Identity fields
     private final TutorialGroup tutorialGroup;
     private final StudentNumber studentNumber;
+    private final ArrayList<Assignment> assignments;
```

Comment:
Technically we could just initialize assignments here to a default empty `ArrayList` that way we don't have to add a new constructor parameter. This is also cause I think every student will be initialized an empty `ArrayList` from my tracing of the code. Unless, you would like to allow the initialise students with assignments then we can just stick to this.

Reply:
Ok agreed. Fixed in latest commit